Detections/Corelight/CorelightMultipleCompressedFilesTransferredOverHTTP.yaml

```diff
@@ -0,0 +1,32 @@
+id: 4e55e306-3022-43a1-870a-41c4d5116079
+name: Corelight - Multiple Compressed Files Transferred over HTTP
+description: |
+  'Detects compressed archives transferre over HTTP.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated